.github/workflows/lint.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rust_version: ["1.76.0", "stable", "nightly-2024-10-21"]
+        rust_version: ["1.78.0", "stable", "nightly-2024-10-21"]
```

Comment:
we should probably update the nightly as well as we tend to only do that when we bump the MSRV.